go/core.rst

```diff
@@ -0,0 +1,425 @@
+
+**TODO**: More information
+
+API
+---
+
+go_library
+~~~~~~~~~~
+
+This builds a Go library from a set of source files that are all part of
+the same package.
+
+Providers
+^^^^^^^^^
+
+* GoLibrary_
+* GoEmbed_
+
+Output groups
+^^^^^^^^^^^^^
+
+* default *: A library with the default build options.*
+* race_ *: The library build with race detection enabled.*
+
+Attributes
+^^^^^^^^^^
+
++----------------------------+-----------------------------+---------------------------------------+
+| **Name**                   | **Type**                    | **Default value**                     |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`name`              | :type:`string`              | |mandatory|                           |
++----------------------------+-----------------------------+---------------------------------------+
+| A unique name for this rule.                                                                     |
+|                                                                                                  |
+| To interoperated cleanly with gazelle_ right now this should be :value:`go_default_library`.     |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`importpath`        | :type:`string`              | :value:`""`                           |
++----------------------------+-----------------------------+---------------------------------------+
+| The import path of this library. If unspecified, the library will have an implicit               |
+| dependency on ``//:go_prefix``, and the import path will be derived from the prefix              |
+| and the library's label.                                                                         |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`srcs`              | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| The list of Go source files that are compiled to create the package.                             |
+| Only :value:`.go` files are permitted, unless the cgo attribute is set, in which case the        |
+| following file types are permitted: :value:`.go, .c, .s, .S .h`.                                 |
+| The files may contain Go-style `build constraints`_.                                             |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`deps`              | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| List of GoLibrary_ providers this library imports directly.                                      |
```

Comment:
I would rephrase this as "List of Go libraries this library imports directly. These may be `go_library` rules or compatible rules with the `GoLibrary` provider."

Reply:
Done